src/static/app/index.css

```
@@ -1,35 +1,93 @@
 /* ======================================================
-  common.
+  common
 =======================================================*/
+
+/* DBFlute-like background */
 body {
   background: transparent url(../image/parts/background.png) repeat-x;
 }
 
+h1 {
+	margin: 0 -20px 40px -20px;
+	padding: 40px 0 10px 140px;
```

```suggestion
	margin: 0 0 40px 0;
	padding: 40px 20px 10px 140px;
```
avoid unnecessary negative margin.

---

Thanks, fixed it